src/server/templates/page.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Render a full HTML page
+ * @param {Object} params
+ * @param {?string} params.title Title of the page
+ * @param {string} [params.body = ""] HTML to render in the body of the page
+ * @param {Object.<string, Object.<string, string>>} params.assets Manifest of filename entry points to bundled assets.
+ * @return {string} HTML
```

Comment:
I find that marking the nullity expectations to greatly enhance documentation. Let's talk about we want to annotate future JSDocs!